Nos/Views/Components/Form/NosTextEditor.swift

```diff
@@ -3,11 +3,16 @@ import SwiftUINavigation
 
 struct NosTextEditor: View {
     
-    var label: LocalizedStringResource
+    var label: LocalizedStringKey
     @Binding var text: String
     
+    init(_ label: LocalizedStringKey, text: Binding<String>) {
+        self.label = label
+        self._text = text
+    }
```

Comment:
I did that in order to match the SwiftUI APIs (`Text`, `Button`, etc.) so that you didn't need the argument label "label:" when using a `LocalizedStringKey`.

That way call sites will look like:
```swift
NosTextEditor("bio")
```

rather than:
```swift
NosTextEditor(label: "bio")
```

Reply:
Awesome. Thanks for the explanation.